src/v2/client/startLoginFlow.ts

```diff
@@ -33,6 +34,11 @@ export async function startLoginFlow(settings) {
     shouldProceedWithEmailAuthenticator: false, // do not auto-select email authenticator
   };
 
+  authClient.http.setRequestHeader("x-okta-amr-hint", "{\"type\":\"pop\",\"hint\":{\"supported\":"
```

Comment:
`webauthn.isPlatformAuthenticatorSupported()` is async call, you might need to await for this